res/css/views/messages/_CreateEvent.scss

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 New Vector Ltd
+Copyright 2020 The Matrix.org Foundation C.I.C.
```

Comment:
The 2018 year is retained:
```suggestion
Copyright 2018, 2020 The Matrix.org Foundation C.I.C.
```

Reply:
This file was deleted and recreated in reality.

Reply:
The copyright is still maintained regardless. Unless you somehow wrote all the CSS exactly the same as what was there before without using the old file as reference.

Reply:
Yeah, that's exactly what happened... 

![image](https://user-images.githubusercontent.com/2403652/98264682-10057480-1f80-11eb-9fd3-99baa089af43.png)

Reply:
The year still transfers: we were supposed to re-assign it to the foundation ages ago but forgot/haven't gotten around to it. At the time the file was edited, it would have been implicitly `2018, 2020` - just making that clearer. The file also wasn't deleted and recreated as far as git is concerned, so the copyright needs to be retained.